compose/ui/ui/src/desktopTest/kotlin/androidx/compose/ui/awt/ComposeFocusTest.kt

```diff
@@ -718,8 +717,6 @@ class ComposeFocusTest {
 
 fun runFocusTest(action: suspend FocusTestScope.() -> Unit) {
     Assume.assumeFalse(GraphicsEnvironment.getLocalGraphicsEnvironment().isHeadlessInstance)
-    // on macOs if we run all tests, the window starts unfocused, so tests fail
-    Assume.assumeFalse(hostOs.isMacOS)
```

Comment:
Does this work when we run all tests? It did work before for an individual run, but didn't when we run all tests.

Probably it is because we didn't close windows back then, and now we close.

Reply:
It does now that I've fixed `ComposeFocusTest` to send `KEY_RELEASED`.

Reply:
It didn't worked before because windows became out of focus, and key events didn't dispatch to the window.

But let's see how it works now, and merge. If we'll encounter any issues, we'll decide later what to do.